realm/realm-library/src/main/java/io/realm/valuelist/RealmValueListQuery.java

```diff
@@ -0,0 +1,444 @@
+
+    public RealmValueListQuery<E> isNotNull() {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable String value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable String value, Case casing) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Long value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Integer value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Short value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Byte value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Double value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Float value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Boolean value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> equalTo(@Nullable Date value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(String[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(String[] values, Case casing) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Long[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Integer[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Short[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Byte[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Double[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Float[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Boolean[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> in(Date[] values) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> notEqualTo(@Nullable String value) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> notEqualTo(@Nullable String value, Case casing) {
+        // TODO implement this
+        return null;
+    }
+
+    public RealmValueListQuery<E> notEqualTo(@Nullable Long value) {
```

Comment:
I actually feel supporting `short`/`int`/`byte` was an incorrect design ... it adds too much methods but not much benefit for users :( `long` only would be good enough for all cases. But it seems we need to follow it.